### DISCUSSION TITLE
Question: This is more of a conversation starter than a question about the Citation field on the Multimedia website. [RW]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#617] [Q&A]
Description: How do people feel about the citation field? 
 
There is a Field Museum website page on preferred citations (https://www.fieldmuseum.org/science/research/use-collections-data-and-images/preferred-citations-collections-data-and-images)
 
Examples: https://mm.fieldmuseum.org/ca9c9874-9acd-4141-ade9-05dc96d90626
https://mm.fieldmuseum.org/f2b2c7a2-a9b8-4ee0-91c0-dfabc04396ef
	
Suggestion(s): 

[NC]
Copyright statement/holder first? – if any
License, if any
Identifier (GUID)
Creator(s)
Language edit – “Courtesy” vs (?)

[ED]	
for anthro image requests the formula we use is (c) Field Museum, Image No (Photo Archives image #), Cat No. Photographer (if the Field Museum staff took the photo)
GUID

ContentDM for examples of different styles?
Offers Chicago/APA etc formats for citations

The citation field in multimedia is a single field but, reports could show more values and also look at displaying different styles on the mm pages? Of course this is all dependent on fields being filled in in the record itself.

Also need to account for “flat” art that has been photographed.
Where you need to credit the artist first then make sure that that party record is attached first in the list.

**Comment by ElanaDux** [comment]
Anthro: © The Field Museum, Image No. [photo archives # if there is one], Cat. No., Photographer Name (if FM provided image) 
Courtesy of The Field Museum, Cat. No., Photographer (if researcher is using their own image of an FM object)
